### ISSUE TITLE
An error will occur if batch_2 is greater than 1

### ISSUE THREAD

**Issue by MetaInsight7** [FlyMyAI/flymyai-lora-trainer#19]
When batch size is greater than 1, an error will occur because the image is of mixed resolution, resulting in torch.stack error. How can we consider solving this problem? Is there a unified padding?
<img width="2090" height="400" alt="Image" src="https://github.com/user-attachments/assets/b84fbc77-4d66-422b-85ad-5cbe6a3484d9" />
